lib/types.ts

```diff
@@ -0,0 +1,11 @@
+import type { AssetProps, ContentTypeProps, EditorInterfaceProps, EntryProps, LocaleProps, TagProps, WebhookProps } from 'contentful-management'
+
+export type SpaceResources = {
```

Comment:
To me they look more like environment resources 🤔

Reply:
That was also my first thought, [but webhooks aren't environment aware](https://www.contentful.com/faq/environments/#what-entities-can-be-copied-to-a-space-environment). Given the time this tool was created (before the introduction of environments) my suggestion is to stick with the jargon of that time  and only if we actually restructure the logic also rename.

Reply:
done ✅